scripts/config_exadg.sh

```diff
@@ -37,6 +37,8 @@ FFTW=$WORKING_DIRECTORY/fftw/install
 LIKWID=$WORKING_DIRECTORY/likwid/install
 
 cmake \
+    -D CMAKE_BUILD_TYPE="DebugRelease" \
+    -D CMAKE_C_FLAGS="-march=native -Wno-array-bounds" \
```

Comment:
This relates to the deal.II setup script. I suppose CXX should work as well. These lines of code become relevant since the deal.II macro does not take care about it anymore.